game_event_player/tests.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib.auth import get_user_model
+import pytest
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+from .models import GameEventPlayer
+from game_event.models import GameEvent
+from player.models import Player, BallGame
+from court.models import Court
+from decimal import Decimal
+from datetime import datetime
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def saved_player(user):
+    player = Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def saved_court():
+    court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                 city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+    return court
+
+
+@pytest.fixture
+def saved_game_event(saved_court):
+    game_event = GameEvent.objects.create(id=2, time=datetime.today(), level_of_game=3,
+                                          min_number_of_players=2, max_number_of_players=5, court_id=saved_court,
+                                          ball_game='Basketball')
+    return game_event
+
+
+@pytest.fixture
+def saved_game_event_player(saved_game_event, saved_player):
+    return GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                          ball_responsible=False)
+
+
+@pytest.mark.django_db
+class TestGameEventPlayerModel:
+
+    def test_game_event_player_creation(self, saved_game_event, saved_player):
+        game_event_player = GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                                           ball_responsible=False)
+        assert game_event_player.game_event_id == saved_game_event
+        assert game_event_player.player_id == saved_player
+        assert game_event_player.ball_responsible is False
+
+    def test_unique_together_constraint(self, saved_game_event, saved_player):
+        GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player, ball_responsible=False)
+        with pytest.raises(IntegrityError):
+            GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                           ball_responsible=False)
```

Comment:
I'd add a positive test case in which you create 3 players for one event and also add 2 of them to another event, and then do some filtering: seeing all events for a specific players, and all the players of a specific event.